woocommerce/woocommerce-android
# [Bug] Fix for order taxes total calculation

Fix for order taxes total calculation bug

### Description

As reported by the users and confirmed in our internal research, there is a bug in order creation flow resulting in wrong taxes total calculation in case the merchant has the "Prices entered with tax" set to "Yes, I will enter prices inclusive of tax"

Context and discussion – peaMlT-kn-p2

### Testing instructions

1. Set "Prices entered with tax" setting to "Yes, I will enter prices inclusive of tax" 
2. Ensure you have at least one tax rate defined
3. Create order and add item(s) to it
4. Verify that the taxes are calculated correctly
